test/development/app-dir/owner-stack/owner-stack.test.ts

```diff
@@ -172,9 +172,9 @@ describe('app-dir - owner-stack', () => {
     await expect(browser).toDisplayCollapsedRedbox(`
      {
        "count": 1,
-       "description": "string in rejected promise",
+       "description": "Error: string in rejected promise",
        "environmentLabel": null,
-       "label": "Console Error",
+       "label": "Runtime Error",
```

Comment:
The label change is a bug fix. Promise rejections shouldn't be treated as console errors. The "Error:" prefix in the description is unfortunate but it goes hand in hand with fixing `reportError(string)` actually showing the string instead of an empty description.